web/src/components/onboarding/OnboardingForeignBusinessType.tsx

```diff
@@ -74,13 +85,14 @@ export const OnboardingForeignBusinessType = ({ headerAriaLevel = 2 }: Props): R
         </FormControl>
       </div>
 
-      {state.profileData.foreignBusinessType !== undefined && (
-        <Alert variant="info">
-          <Content>
-            {Config.profileDefaults.FOREIGN.foreignBusinessType[state.profileData.foreignBusinessType]}
-          </Content>
-        </Alert>
-      )}
+      {state.profileData.foreignBusinessType !== undefined &&
+        state.profileData.foreignBusinessType !== "NONE" && (
```

Comment:
this is a stylistic preference, but personally, when I start to have more than a single item in an inline JSX boolean check (like this), I like to extract it out into a function to make it more readable. Something like `shouldShowForeignBusinessTypeAlert` or something. Prevents the JSX from getting too wordy with inline javascript logic checks